library/core/src/future/future.rs

```diff
@@ -103,6 +105,38 @@ pub trait Future {
     #[lang = "poll"]
     #[stable(feature = "futures_api", since = "1.36.0")]
     fn poll(self: Pin<&mut Self>, cx: &mut Context<'_>) -> Poll<Self::Output>;
+
+    /// Map this future's output to a different type, returning a new future of
+    /// the resulting type.
+    ///
+    /// This function is similar to [`Option::map`] or [`Iterator::map`] where
+    /// it will change the type of the underlying future. This is useful to
+    /// chain along a computation once a future has been resolved.
+    ///
+    /// Note that this function consumes the receiving future and returns a
+    /// wrapped version of it, similar to the existing `map` methods in the
```

Comment:
```suggestion
    /// wrapped version of it, similarly to the existing `map` methods in the
```

Reply:
@mcol I may be misparsing, but I don't think that's a grammatically correct change; I think this was correct as written.